### PR TITLE
Create Broadcast Page

### PR DESCRIPTION
**Pull request by JLopezz3**

### CODE REVIEW

File: assets/Broadcast_page/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# BroadCast Page created by JLopezz3
```
Comment (Kajol-Kumari) [inline assets/Broadcast_page/README.md:1]: can you please add the figma link in here?

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]

File: assets/Broadcast_page/README.md (hunk 1 above)
Comment (JLopezz3) [reply assets/Broadcast_page/README.md:1]: Okay

Comment (JLopezz3) [reply assets/Broadcast_page/README.md:1]: Done. I added the Figma link in the README

**Review by Kajol-Kumari — APPROVED** [review]
LGTM :+1:

**Pull request merged**